samples/ResourceOwnerPasswordFlow/Certificate.cer

```diff
@@ -0,0 +1,24 @@
+Bag Attributes
```

Comment:
Okay. Just note that you'll have to add the (ugly) workarounds I used for Mvc.Server if you want to support Core CLR and Mono.